src/index.html

```diff
@@ -17,16 +17,30 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form data-qa="big">
+      <div class="search__big">
+        <div class="search__big-icon">
+          <input
+            class="search__big-input"
+            type="text"
+            data-qa="keypress"
+            placeholder="Try “Los Angeles“"
+          />
+        </div>
+      </div>
+    </form>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form data-qa="small">
+      <div class="search__small">
+        <div class="search__small-icon">
+          <input
+            class="search__small-input"
+            type="text"
+            data-qa="hover"
+            placeholder="Try “Los Angeles“"
+          />
```

Comment:
Similar to the previous input, consider adding a 'name' attribute to this input element for better form data handling.